src/NetscriptJSEvaluator.ts

```diff
@@ -160,7 +158,12 @@ function generateLoadedModule(script: Script, scripts: Map<ScriptFilePath, Scrip
     // servers; it will be listed under the first server it was compiled for.
```

Comment:
Oh lol, actually this comment and code is what I was thinking of. I guess I *did* do something with this at some point.